kolibri/core/assets/src/views/ExamReport/__tests__/AttemptIconDiff.spec.js

```diff
@@ -0,0 +1,38 @@
+import { render, screen } from '@testing-library/vue';
+import '@testing-library/jest-dom';
+import VueRouter from 'vue-router';
+import AttemptIconDiff from '../AttemptIconDiff.vue';
+
+const successThemeColor = '#43a047';
```

Comment:
In te future we can change theme colors, so it is better to use the themeTokens variables, you can import it like this:

```js
import { themeTokens } from 'kolibri-design-system/lib/styles/theme';

const successThemeColor = themeTokens().correct;
```

Reply:
Ahh, nice catch. Will update!